test_unstructured/partition/test_auto.py

```diff
@@ -775,6 +775,15 @@ def test_auto_partition_csv_from_file(filename="example-docs/stanley-cups.csv"):
     assert elements[0].metadata.filetype == "text/csv"
 
 
+def test_auto_partition_html_pre_from_file(filename="example-docs/fake-html-pre.htm"):
+    elements = partition(filename=filename)
+
+    assert len(elements) > 0
```

Comment:
maybe slightly more sanity testing on the `.text` content of the element(s) here?